src/main/java/core/basesyntax/model/zoo/Cat.java

```diff
@@ -1,5 +1,8 @@
 package core.basesyntax.model.zoo;
 
+import javax.persistence.Entity;
+
+@Entity
```

Comment:
since you have added DiscriminatorColumn lets add DiscriminatorValue to child classes

Reply:
I've just decided to keep the class name as it is but ok :)